<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputData</class>
 <widget class="QWidget" name="InputData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>966</width>
    <height>856</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>966</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ввод исходных данных</string>
  </property>
  <widget class="QListWidget" name="resultList">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>550</y>
     <width>931</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="propertyList">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>431</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="viewKnowledgeBase">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>820</y>
     <width>211</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Посмотреть базу знаний</string>
   </property>
  </widget>
  <widget class="QPushButton" name="determineTypeAlloy">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>820</y>
     <width>201</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Определить вид сплава</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>480</width>
     <height>380</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>470</width>
     <height>380</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>550</width>
     <height>390</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>550</width>
     <height>390</height>
    </size>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>457</width>
      <height>378</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>551</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="listWidget_4">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ввод исходных данных</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>490</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введенные значения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearInputValues">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>820</y>
     <width>201</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Очистить значения</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
